packages/cycle-scripts-ts-one-fits-all/scripts/init.js

```diff
@@ -0,0 +1,192 @@
+  // Rename gitignore after the fact to prevent npm from renaming it to .npmignore
+  // See: https://github.com/npm/npm/issues/1862
+  var gitignorePath = path.join(appPath, 'gitignore')
+  var dotGitignorePath = path.join(appPath, '.gitignore')
+  fs.move(gitignorePath, dotGitignorePath, [], function (err) {
+    if (err) {
+      // Append if there's already a `.gitignore` file there
+      if (err.code === 'EEXIST') {
+        var content = fs.readFileSync(gitignorePath)
+        fs.appendFileSync(dotGitignorePath, content)
+        fs.unlinkSync(gitignorePath)
+      } else {
+        throw err
+      }
+    }
+  })
+}
+
+function patchIndexTs (appPath, tags) {
+  var indexTsPath = path.join(appPath, 'src', 'index.ts')
+  var content = fs.readFileSync(indexTsPath, {encoding: 'utf-8'})
+  fs.writeFileSync(
+    indexTsPath,
+    replaceTags(content, tags)
+  )
+}
+
+function patchAppTs (appPath, tags) {
+  var indexTsPath = path.join(appPath, 'src', 'app.ts')
+  var content = fs.readFileSync(indexTsPath, {encoding: 'utf-8'})
+  fs.writeFileSync(
+    indexTsPath,
+    replaceTags(content, tags)
+  )
+}
+
+function successMsg (appName, appPath) {
+  console.log()
+  console.log('Success! Created ' + appName + ' at ' + appPath)
+  console.log('Inside that directory, you can run several commands:')
+  console.log()
+  console.log(chalk.cyan('  npm start'))
+  console.log('    Starts the development server')
+  console.log()
+  console.log(chalk.cyan('  npm test'))
+  console.log('    Start the test runner')
+  console.log()
+  console.log(chalk.cyan('  npm run build'))
+  console.log('    Bundles the app into static files for production')
+  console.log()
+  console.log(chalk.cyan('  npm run eject'))
+  console.log('    Removes this tool and copies build dependencies, configuration files')
+  console.log('    and scripts into the app directory. If you do this, you can\'t go back!')
+  console.log()
+  console.log('We suggest that you begin by typing:')
+  console.log()
+  console.log(chalk.cyan('  cd ' + appName))
+  console.log(chalk.cyan('  npm start'))
+  console.log()
+  console.log('If you have questions, issues or feedback about Cycle.js and create-cycle-app, please, join us on the Gitter:')
+  console.log()
+  console.log(chalk.cyan('  https://gitter.im/cyclejs/cyclejs'))
+  console.log()
+  console.log('Happy cycling!')
+}
+
+module.exports = function (appPath, appName, streamLib, verbose, originalDirectory) {
+  var ownPackageName = require(path.resolve(__dirname, '..', 'package.json')).name
+  var ownPath = path.join(appPath, 'node_modules', ownPackageName)
+  var appPackageJson = path.join(appPath, 'package.json')
+  var appPackage = require(appPackageJson)
+
+  // Manipulate app's package.json
+  appPackage.dependencies = appPackage.dependencies || {}
+  appPackage.devDependencies = appPackage.devDependencies || {}
+  appPackage.scripts = {
+    'start': 'NODE_ENV=development webpack --config webpack.config.js',
```

Comment:
Should I use a .babelrc file in the template or should I modify the package.json?